### PR TITLE
Refactoring Places

### PR DESCRIPTION
**Pull request by saarSAARsaar**
Made the root places_index and made appropriate changes from there

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,8 +1,9 @@
 Rails.application.routes.draw do
-  get 'places/:id', to: 'places#show', as: 'places'
+  #get 'places/:id', to: 'places#show', as: 'places'
```
Comment (ahorincar) [inline config/routes.rb:2]: I guess we can delete this line altogether.

**Review by ahorincar — COMMENTED** [review]
Great job on your first refactoring session! I left a few minor comments, mostly regarding the coding style.

Also, I think we have a mix of 2-spaces and 4-spaces indentation in some of these files. I would suggest sticking with the 2-spaces indentation. I think the editor should have an option to set the default indentation to 2 spaces.

If you have any doubts or questions, let me know. in comments maybe. Otherwise, if you agree with my suggestions, you could make the changes and push a new version on this branch and I will have another look afterwards.

**Comment by saarSAARsaar** [comment]
Did as said above

**Review by ahorincar — APPROVED** [review]
Nice! ⭐

**Pull request merged**